eng/CredScanSuppression.json

```diff
@@ -43,7 +43,8 @@
                 "Yukon900!",
                 "sql@zure123!",
                 "asdsadsad",
-                "Kg=="
+                "Kg==",
+                "jQetX8odiJoZ7Yo0X8vWgh%2FMqRv9WE3GU%2Fr%2BLNMK3GU%3D"
```

Comment:
Can we reuse one of the existing placeholders?

Reply:
Given that Event Hubs is already using
https://github.com/Azure/azure-sdk-for-net/blob/3b9003dbe08a60ef83c94eceb6ab9ae1c7c3dc35/eng/CredScanSuppression.json#L7
we can try updating the affected tests to use the same secret. Do you think we can do that without compromising testability, @christothes ?

After testing I noticed the same secret is used in 4 other storage files, so we may need to update them as well:
https://github.com/Azure/azure-sdk-for-net/blob/3b9003dbe08a60ef83c94eceb6ab9ae1c7c3dc35/sdk/storage/Azure.Storage.Blobs/tests/BlobUriBuilderTests.cs#L725
https://github.com/Azure/azure-sdk-for-net/blob/3b9003dbe08a60ef83c94eceb6ab9ae1c7c3dc35/sdk/storage/Azure.Storage.Files.DataLake/tests/DataLakeUriBuilderTests.cs#L389
https://github.com/Azure/azure-sdk-for-net/blob/3b9003dbe08a60ef83c94eceb6ab9ae1c7c3dc35/sdk/storage/Azure.Storage.Files.Shares/tests/FileUriBuilderTests.cs#L259
https://github.com/Azure/azure-sdk-for-net/blob/3b9003dbe08a60ef83c94eceb6ab9ae1c7c3dc35/sdk/storage/Azure.Storage.Queues/tests/QueueUriBuilderTests.cs#L443

Reply:
If the value needs to be a valid SAS, I doubt using the Event Hubs secret will work.   The messaging services have their own unique format.   For that to have a chance to work, we'd need to use the same key name and key value that the Event Hubs tests were using.

Reply:
The idea is to try and reduce the number of unique fake secrets we need to keep track of in the repo so if we can come up with a valid SAS that would work in multiple context that would be idea. If there are unique and cannot be shared then I guess we will have to do what we have to.